docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,386 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
 $ sensorctl test accelerometer 100 1000 0 /* enable accelerometer with interval 100 ms, 1s batch latency and always on option */
 $ sensorctl info accelerometer /* retrieve accelerometer sensor information */
 ```
+
+## Crash Framework
+
+### crash-manager
+
+crash-manager is a tool used to create crash reports when an application gets crashed.
+crash-manager is not usually invoked by manually, it is invoked automatically by the kernel when the crash occurs. Crash-manager generates the crash report and attaches the kernel-provided core, if requested. Crash-worker package provides the `99-crash-manager.conf` file, which configures kernel parameters for crash-manager to get invoked automatically on crash. For more information, see [`99-crash-manager.conf`](#99-crash-managerconf).
+
+crash-manager consists of the following options:
+
+Option                | Description
+----------------------|------------------------------------------------------
+-p  --pid=PID         | Specifies the process ID (PID) of the dumped process
+-u  --uid=UID         | Specifies the real user identifier (UID) of the dumped process
+-g  --gid=GID         | Specifies the real group identifier (GID) of the dumped process
+-i  --tid=TID         | Specifies the thread identifier (TID) of the thread that triggered core dump. For more information, see [core(5)](http://man7.org/linux/man-pages/man5/core.5.html).
```

Comment:
Global: As suggest by @mijin85cho, please add the period everywhere at the end of the sentence.

Reference: https://github.com/Samsung/tizen-docs/pull/791#discussion_r314591602

@mijin85cho, please verify.

Reply:
@mijin85cho,

Thank you for your response.